modules/ROOT/pages/_partials/sync/js-client-realtime-updates.adoc

```diff
@@ -1,3 +1,63 @@
 = Realtime Updates
 
-TODO
\ No newline at end of file
+Subscriptions are a core concept of GraphQL and one which allow us to enable real time updates and synchronisation of data.
+GraphQL Subscriptions use web sockets to enable clients to subscribe to published changes. {sync-client} comes with websocket and therefore
+subscription support out of the box. To enable subscriptions on your client set the following paramater in the {sync-client} config object.
+A DataSyncConfig interface is also available from {sync-client} if you wish to use it.
+
+An example of configuring the client for subscriptions can be seen below.
+
+```
+const config: DataSyncConfig {
+    ...
+    wsUrl: { WEB_SOCKET_URL_HERE }
+    ...
+}
+```
+Where wsUrl is the full URL of the websocket endpoint of your GraphQL server. Once this object is created, use it to initialise {sync-client} with:
```

Comment:
this is about manually setting the `wsUrl`, right? Can we also mention that if you are using `mobile-services.json` file, it is setup automatically?